.github/workflows/main.yml

```diff
@@ -26,17 +26,15 @@ jobs:
           php-version: ${{ matrix.php }}
 
       - name: Restore Composer package cache
-        id: composer-cache
+        id: composer-cache-root
```

Comment:
I'm not sure what you intended to achieve by changing this

Reply:
Oh, I was trying to invalidate the original cache, but turns out it is not necessary anymore and I forgot to revert this.